components/gantry/singleaxis/singleaxis.go

```diff
@@ -563,5 +591,6 @@ func (g *singleAxis) CurrentInputs(ctx context.Context) ([]referenceframe.Input,
 func (g *singleAxis) GoToInputs(ctx context.Context, goal []referenceframe.Input) error {
 	g.mu.Lock()
 	defer g.mu.Unlock()
-	return g.MoveToPosition(ctx, referenceframe.InputsToFloats(goal), nil)
+	speed := []float64{g.rpm}
```

Comment:
debug statement.

Reply:
so this one I'm a little confused about because `GoToInputs` doesn't take a speed, so I don't know if we should add a debug statement saying we are using the default speed because that's the only option. So I think every time you used `++` or `--` on app you would see the debug statement which doesn't really seem applicable. I could pass in `[]float64{}` to `MoveToPosition` so that debug statement will be printed there? But still kind of the same reasoning that `GoToInputs` doesn't take a speed so should we be logging anything about speed?

Reply:
GoToInputs isn't exposed in the API. This gets passed in from a `motion.Move` call which is exposed in out API. The GoToInputs function signature can also change in the future. 

That being said I think you're right, let's just let MoveToPosition handle all the floats passed in cases, and pass in an empty float here. If it's empty in MoveToPosition, the scope do specifies using the default gantry speed.